mapserver/django/django_project/fba/management/commands/ingest_boundary_data.py

```diff
@@ -28,88 +30,168 @@ def is_table_exists(self, conn, table_name, schema_name):
         is_exist = cur.fetchone()
         return is_exist[0]
 
-    def get_table_header(self, conn, table_name):
-        cur = conn.cursor()
-        sql = "SELECT * FROM {table_name} WHERE 1=0".format(
-            table_name=table_name
-        )
-        cur.execute(sql)
-        table_header = [d[0] for d in cur.description]
-        cur.close()
-        return table_header
+    def get_table_header(self, conn, schema_name, table_name):
+        try:
+            with conn.cursor() as cur:
+                sql = f'SELECT * FROM {schema_name}.{table_name} LIMIT 1'
+                cur.execute(sql)
+                return [d[0] for d in cur.description]
+        except:
+            return []
 
-    def get_admin_data(self, conn, table_name):
-        cur = conn.cursor()
-        sql = "SELECT * FROM {table_name}".format(
-            table_name=table_name
-        )
-        cur.execute(sql)
-        rows = cur.fetchall()
-        cur.close()
-        return rows
+    def get_admin_data(self, conn, schema_name, table_name):
+        try:
+            with conn.cursor() as cur:
+                sql = f'SELECT * FROM {schema_name}.{table_name}'
+                cur.execute(sql)
+                return cur.fetchall()
+        except:
+            return []
 
     def create_connection(self):
         try:
-            conn = driv.connect(database='gis',
-                                user='docker',
-                                password='docker',
-                                host='192.168.1.197',
-                                port='45432')
+            conn = driv.connect(database='geocris',
+                                user='cdema',
+                                password='cdema',
+                                host='192.168.100.6',
+                                port='35432')
```

Comment:
@dimasciput said we just use hardcoded local value for now because it will be changed later to use pg_featureserv.